es/README.md

```diff
@@ -2,21 +2,19 @@
 
 ### Manual installation
 
-Download Elasticsearch from https://www.elastic.co/fr/downloads/elasticsearch
-and copy it to the ES module. eg. es/elasticsearch-5.0.0
+Download Elasticsearch (at least 7.2.1 for Geonetwork 3.8.x) from https://www.elastic.co/fr/downloads/elasticsearch
+and copy it to the ES module. eg. es/elasticsearch-7.2.1
```

Comment:
Maybe we can even remove this. It can be installed in other location or machine.

Reply:
@fxprunayre I think we should still include this, but if there are ways of using ES on a remote machine then we should also document that (I presume you just change the kibana config to point at the remote index?)

Reply:
> (I presume you just change the kibana config to point at the remote index?)

yes.